test/JDBC/expected/TestDecimal-vu-verify.out

```diff
@@ -265,3 +265,11 @@ select in3 * in4 from testdecimal_vu_prepare_tab2;
 
 ~~ERROR (Message: Arithmetic overflow error for data type numeric.)~~
 
```

Comment:
Could you add a test case like 

select from a join b on cast (a.t as real) = CAST(CAST(b.t  AS real) AS decimal(30,2))

something like that

thanks